### DISCUSSION TITLE
The formula of  model

### DISCUSSION THREAD

**Discussion by Supercn01** [Open-Systems-Pharmacology/Forum#1607] [Questions & Problems] [answered]
Hi all

   I have developed a PBPK model. I wonder how check the formula of the model. Could anyone tell me how check the formula? 

Thanks

**Comment by nina-nauwelaerts** [comment, marked as answer]
You can right click on the simulation and then select "Export Simulation Structure to File". This creates a text file with the parameters and formulas. Or you could open the model in Mobi to see the formulas there.
